pkg/network/driver_interface.go

```diff
@@ -359,11 +342,10 @@ func (di *DriverInterface) GetOpenConnectionStats(openBuf *ConnectionBuffer, fil
 	if err != nil {
 		return 0, err
 	}
-	di.openFlows.Add(int64(count))
-	di.totalFlows.Add(int64(count))
-
-	di.nOpenBufferIncreases.Add(int64(increases))
-	di.nOpenBufferDecreases.Add(int64(decreases))
+	driverTelemetry.openFlows.Add(float64(count))
+	driverTelemetry.totalFlows.Add(float64(count))
```

Comment:
I realize this is just replicating the previous behaviour, but the previous behaviour was flawed... would you mind changing this so that we just set the telemetry value to `count` here, that way we don't need periodically reset these value in `RefreshStats`?

Reply:
sure that makes sense to me!

Reply:
for posterity I changed `openFlows` and `closedFlows` to be Gauges that are set instead of incremented and left `totalFlows` as a Counter that will monotonically increase. I removed `moreDataErrors` as it wasn't being incremented anywhere

Reply:
That's perfect, thank you!